packages/ag-grid-theme/css/_uitk-ag-theme-mixin.scss

```diff
@@ -571,7 +571,6 @@
 
   [class^="ag-"]:focus {
     outline: none;
-    box-sizing: border-box;
```

Comment:
@ararus Do these changes make sense to you?

Reply:
It should be fine to move it to global, looks ok